README.md

```diff
@@ -11,6 +11,8 @@
 [![Downloads per month](https://img.shields.io/packagist/dm/sentry/sentry.svg?style=flat-square)](https://packagist.org/packages/sentry/sentry)
 [![Latest stable version](https://img.shields.io/packagist/v/sentry/sentry.svg?style=flat-square)](https://packagist.org/packages/sentry/sentry)
 [![License](http://img.shields.io/packagist/l/sentry/sentry.svg?style=flat-square)](https://packagist.org/packages/sentry/sentry)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/getsentry/sentry-php/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/getsentry/sentry-php/)
```

Comment:
You should use [shields.io](http://shields.io/) like all other badges, and the same for the line below